internal/handlers/handlers.go

```diff
@@ -59,6 +61,13 @@ func GetLotteryNumbers(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// VersionHandler writes the current application version to the response.
+func VersionHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte(version.Version))
```

Comment:
Consider handling the error returned by `w.Write([]byte(version.Version))` in `VersionHandler`. Even though it's unlikely to fail, it's a good practice to handle all errors. [important]